### ISSUE TITLE
error when svg contains unicode

### ISSUE THREAD

**Issue by zhnuaa** [btel/svg_utils#29]
heres's error log:

```python
UnicodeDecodeError                        Traceback (most recent call last)
<ipython-input-15-49ac8e8f02dd> in <module>
      7 sc.Figure('8cm','8cm',
      8           sc.Panel(sc.SVG(r'NACA0012.svg')),
----> 9           sc.Panel(sc.SVG(r'CaseVerify.svg')))

c:\python37\lib\site-packages\svgutils\compose.py in __init__(self, fname)
    108     def __init__(self, fname):
    109         fname = os.path.join(CONFIG['svg.file_path'], fname)
--> 110         svg = _transform.fromfile(fname)
    111         self.root = svg.getroot().root
    112 

c:\python37\lib\site-packages\svgutils\transform.py in fromfile(fname)
    308     fig = SVGFigure()
    309     fid = open(fname)
--> 310     svg_file = etree.parse(fid)
    311     fid.close()
    312 

src\lxml\etree.pyx in lxml.etree.parse()

src\lxml\parser.pxi in lxml.etree._parseDocument()

src\lxml\parser.pxi in lxml.etree._parseFilelikeDocument()

src\lxml\parser.pxi in lxml.etree._parseDocFromFilelike()

src\lxml\parser.pxi in lxml.etree._BaseParser._parseDocFromFilelike()

src\lxml\parser.pxi in lxml.etree._ParserContext._handleParseResultDoc()

src\lxml\parser.pxi in lxml.etree._handleParseResult()

src\lxml\etree.pyx in lxml.etree._ExceptionContext._raise_if_stored()

src\lxml\parser.pxi in lxml.etree._FileReaderContext.copyToBuffer()

UnicodeDecodeError: 'gbk' codec can't decode byte 0xbc in position 4126: illegal multibyte sequence
```
i add `encoding` args to `transform.py` fix the problem:

```bash
309c309
<     fid = open(fname,encoding='utf-8')
---
>     fid = open(fname)
```

**Comment by truefusion** [comment]
Similar fix for `fromstring` would be:
change
`svg = etree.fromstring(text.encode())`
to
`svg = etree.fromstring(text.encode("utf-8"))`

**Comment by btel** [comment]
fixed by #39
